webviz_subsurface/_datainput/well_completions.py

```diff
@@ -1,21 +1,47 @@
-from typing import Optional, Dict
+from typing import Optional, Dict, List, Tuple, Any
 import json
+import re
 from pathlib import Path
 import glob
+import logging
 
-import ecl2df
+from ecl2df import common
+
+
+def remove_invalid_colors(zonelist: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+    """Removes colors in the zonelist from the lyr file that is not 6 digit
+    hexadecimal.
+    """
+    for zonedict in zonelist:
+        if "color" in zonedict and not re.match(
+            "^#([A-Fa-f0-9]{6})", zonedict["color"]
+        ):
+            logging.warning(
```

Comment:
I think the current line will use the `root` logger. :thinking:  Maybe change to the following (consumers can easier turn of logging on module level):
```suggestion
            logging.getLogger(__name__).warning(
```
Ref.: https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial